changelogs/changelog.yaml

```diff
@@ -1971,3 +1971,59 @@ releases:
           created using the same name as an existing uplink, then the existing
           uplink is modified.
     release_date: '2024-07-31'
+
+  9.6.0:
+    changes:
+      release_summary: '- The ``ome_application_certificate`` module is
+          enhanced to support the upload of certificate chain.
+
+        - The ``idrac_support_assist`` module is added to add support
+          to run and export SupportAssist collection logs on iDRAC.
+
+        - The ``idrac_secure_boot`` module is added to import the secure
+          boot certificate.'
+
+      major_changes:
+        - idrac_support_assist - This module allows to run and export
+          SupportAssist collection logs on iDRAC.
+        - idrac_secure_boot - This module allows to import the secure boot
+          certificate.
+
+
+      minor_changes:
+        - ome_application_certificate - This module is enhanced to support
+          the upload of certificate chain.
+
+      known_issues:
+        - idrac_storage_volume - Issue(290766) - The module will report success
+          instead of showing failure for new virtual creation on the BOSS-N1
+          controller if a virtual disk is already present on the same
+          controller.
+        - idrac_diagnostics - Issue(285322) - This module doesn't support
```

Comment:
This module does not support the export of diagnostics files to HTTP and HTTPS share via SOCKS proxy.

Reply:
These are defect title, ignore it.